spec/forms/password_reset_email_form_spec.rb

```diff
@@ -12,29 +12,33 @@
         user = build(:user, :signed_up, email: 'test1@test.com')
         subject = PasswordResetEmailForm.new('Test1@test.com')
 
-        result = {
-          success: true,
-          errors: [],
+        extra = {
           user_id: user.uuid,
           role: user.role,
           confirmed: true,
         }
 
+        result = instance_double(FormResponse)
+
+        expect(FormResponse).to receive(:new).
```

Comment:
We care about the side effects, such as what is sent to analytics right? The particular arguments to this struct don't matter as much? I worry that stubbing `.new` ties this test to the implementation too much